test/runtests.jl

```diff
@@ -1,5 +1,128 @@
-using Distributed, AzManagers, Test
+using Distributed, AzManagers, Random, Test, HTTP, AzSessions, JSON
 
-@testset "AzManagers" begin
-    @test true
+template = "cbox02"
```

Comment:
We will need to run the template create steps here before we run the tests, i believe that @samtkaplan has put the necessary steps in the docs

`myscaleset = AzManagers.build_sstemplate("myvm",
    subscriptionid       = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
    location             = "southcentralus",
    resourcegroup        = "my-resource-group",
    vnet                 = "my-vnet",
    subnet               = "my-vnets-subnet",
    imagegallery         = "my-image-gallery",
    imagename            = "my-image-name",
    skuname              = "Standard_D2s_v3")
AzManagers.save_template_scaleset("myscaleset", myscaleset)`